config/parser.go

```diff
@@ -77,6 +77,8 @@ func (p *Parser) parseLines(lines []string) (err error) {
 			p.opts.logDateTime = parseBool(value, defaultLogDateTime)
 		case "DEBUG":
 			p.opts.debug = parseBool(value, defaultDebug)
+		case "TIMING":
```

Comment:
Can you use a name that is more explicit? For example `SERVER_TIMING_HEADER` or maybe `ENABLE_SERVER_TIMING_HEADER`.

Other than that, looks good to me.

Reply:
Fixed with `SERVER_TIMING_HEADER` :)